source/dbo/Views/Power BI Reporting/vwUnmatchedDuplicates.sql

```diff
@@ -5,40 +5,93 @@ AS
 WITH NtbsNotifications AS (
 	SELECT n.NotificationId AS NtbsId,
 		CAST(n.ETSID AS INT) AS EtsId,
+		'NTBS' AS SourceSystem,
 		n.GroupId AS NtbsGroupId,
 		etsGroupings.GroupId AS EtsGroupId,
 		n.NotificationDate,
 		n.CreationDate,
+		COALESCE(cd.TreatmentStartDate, cd.DiagnosisDate, n.NotificationDate) AS TreatmentStartDate,
+		MAX(te.EventDate) AS LastRecordedTreatmentOutcome,
 		p.Dob,
-		p.FamilyName,
+		CONCAT_WS(', ', UPPER(p.FamilyName), p.GivenName) AS FullName,
+		COALESCE(tbs.[Name], h.[Name]) AS TbServiceName,
+		phec.[Name] AS PhecName,
 		REPLACE(p.NhsNumber, ' ', '') AS NhsNumber,
-		u.Username AS CaseManager
+		COALESCE(u.EmailPrimary, u.Username) AS CaseManager
 	FROM [$(NTBS)].dbo.[Notification] n
 		INNER JOIN [$(NTBS)].dbo.Patients p ON n.NotificationId = p.NotificationId
-		INNER JOIN [$(NTBS)].dbo.HospitalDetails h ON n.NotificationId = h.NotificationId
-		LEFT JOIN [$(NTBS)].dbo.[User] u ON h.CaseManagerId = u.Id
+		INNER JOIN [$(NTBS)].dbo.ClinicalDetails cd ON n.NotificationId = cd.NotificationId
+		INNER JOIN [$(NTBS)].dbo.HospitalDetails hd ON n.NotificationId = hd.NotificationId
+		LEFT JOIN [$(NTBS)].dbo.TreatmentEvent te ON n.NotificationId = te.NotificationId AND te.TreatmentEventType = 'TreatmentOutcome'
+		LEFT JOIN [$(NTBS)].ReferenceData.Hospital h ON hd.HospitalId = h.HospitalId
+		LEFT JOIN [$(NTBS)].ReferenceData.TbService tbs ON h.TBServiceCode = tbs.Code
+		LEFT JOIN [$(NTBS)].ReferenceData.PHEC ON tbs.PHECCode = PHEC.Code
+		LEFT JOIN [$(NTBS)].dbo.[User] u ON hd.CaseManagerId = u.Id
 		LEFT JOIN [$(migration)].dbo.EtsGroupings etsGroupings ON n.ETSID = etsGroupings.EtsId
 	WHERE n.NotificationStatus IN ('Notified', 'Closed')
+	GROUP BY n.NotificationId,
```

Comment:
Is this Group By to get the MAX to come out? I would probably do a PARTITION just to make the code easier to read (or a sub-query)